### ISSUE TITLE
uefi booting

### ISSUE THREAD

**Issue by rmkraus** [project-faros/cluster-manager#38]
Enable UEFI for PXE booting and RHCOS.... I guess 😢

**Comment by rmkraus** [comment]
UEFI PXE Booting:

 - [ ] install grub image
```
yum install grub2-efi-x64.x86_64
cp /boot/efi/EFI/redhat/grubx64.efi /var/lib/tftpboot/
chmod 0644 /var/lib/tftpboot/grubx64.efi
```
 -  [ ] configure dhcpd.conf
```
next-server         192.168.8.1;
filename            "/bootx64.efi";
option space PXE;
option PXE.mtftp-ip    code 1 = ip-address;
option PXE.mtftp-cport code 2 = unsigned integer 16;
option PXE.mtftp-sport code 3 = unsigned integer 16;
option PXE.mtftp-tmout code 4 = unsigned integer 8;
option PXE.mtftp-delay code 5 = unsigned integer 8;
option arch code 93 = unsigned integer 16; # RFC4578
```

**Comment by rmkraus** [comment]
fixed in 0.5.4
